### ISSUE TITLE
indicating sponsorable status

### ISSUE THREAD

**Issue by eshellman** [EbookFoundation/free-programming-books#3447]
GitHub is now allowing Organizations to participate in their sponsor program, so this repo now has a sponsor button benefitting the Free Ebook Foundation.

We have about 300 listing that point to GitHub repos; it seems to me that we could denote those that are sponsorable with a heart emoji: :heart: 

The sponsorable attribute is available in the GitHub v4 API: https://developer.github.com/v4/interface/sponsorable/

Comments? Volunteers?

**Comment by devonzuegel** [comment]
Thanks Eric! We're thrilled to share the extensions to the GitHub Sponsors core we've been working on this fall, so it's nice to hear you're excited about it too.

I'm not sure I quite understand your question—could you rephrase?

**Comment by eshellman** [comment]
I like @devonzuegel 's suggestion to use the :sparkling-heart: instead
